tests/block_tools.py

```diff
@@ -307,14 +310,13 @@ async def new_plot(
             created, existed = await create_plots(
                 args,
                 plot_keys,
-                self.root_path,
                 use_datetime=False,
                 test_private_keys=[AugSchemeMPL.key_gen(std_hash(self.created_plots.to_bytes(2, "big")))],
             )
             self.created_plots += 1
 
             plot_id_new: Optional[bytes32] = None
-            path_new: Optional[Path] = None
+            path_new: Path = Path()
```

Comment:
```suggestion
            path_new: Optional[Path] = None
```

My bad, not sure how i ended up introducing this but its not related here.